src/components/Header/index.jsx

```diff
@@ -150,15 +163,15 @@ const GordonHeader = ({ onDrawerToggle }) => {
           >
             <MenuIcon className={styles.menu_button_icon} />
           </IconButton>
-          &nbsp;&nbsp; &nbsp;&nbsp;
-          <Tab //this is an "invisible" button behind the new logo that links back to the home page
-            className={styles.tab}
-            label=" "
-            component={ForwardNavLink}
-            disableRipple={true}
+          <Link
             to="/"
-            onChange={(event, value) => setTabIndex(value)}
-          />
+            component={ForwardNavLink}
+            value={tabIndex}
+            onClick={(event, value) => setTabIndex(value)}
+          >
+            <img src={logoSizedForHeader()}></img>
```

Comment:
We should use [the `picture` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) instead, which is the browser-supported way to load the best version of an image for the user's device.